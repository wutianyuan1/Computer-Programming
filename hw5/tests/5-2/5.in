2 2 0
abc 12345
cba 54321