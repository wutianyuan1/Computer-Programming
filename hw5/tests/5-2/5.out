/-------------\
| abc | 12345 |
|-----|-------|
| cba | 54321 |
\-------------/
